dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/metadata/WebModuleMetaDataImpl.java

```diff
@@ -169,4 +171,12 @@ public void setAnnotatedSecurityMetaData(Object annotatedSecurityMetaData) {
         
     }
 
+    public boolean checkForHAM() {
+        // Only return false if it has been checked and found to be false.
+        return hasHAM == Boolean.FALSE;
```

Comment:
```suggestion
        return hasHAM != Boolean.FALSE;
```

Reply:
Oooh yes ... annoying that I missed that the first time. Done